container.go

```diff
@@ -97,6 +97,16 @@ func (c *ContainerConfig) valid() bool {
 	return true
 }
 
+// FsmapEntry describes a filesytem map entry related to a container.
+// This needs to be sent to the agent to mount devices/files at the specified location.
+type FsmapEntry struct {
+	Source       string
+	Dest         string
+	ReadOnly     string
+	Permissions  string
+	AbsolutePath bool
+}
```

Comment:
This structure is quite hyperstart specific. Can't we re-use the `Mount` structure and infer the `AbsolutePath` value from it ?

Reply:
@amshinde yes I agree with @sameo. You should keep the `Container` structure unchanged, and fill the `FsmapDescriptor` structure from hyperstart implementation.

Reply:
@sameo  I didnt want to pollute  the `mounts` provided in the configuration with the ones that we need to create for handling `devices`. I would like to keep a distinction between them, there is still some work pending in handling all the mounts besides bind mounts in container `Mount` list. We also need to pass and handle `Permissions` for block devices.
I could rename this struct to DeviceMap to make it less hyperstart specific. What do you think?